teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown')

```

Comment:
- [ ] The same as above you need to add `parent_permission` to your initializer and super.